src/components/ui-shell/_theme.scss

```diff
@@ -0,0 +1,33 @@
+// Header bar background
+$shell-header-bg-01: #252525;
```

Comment:
@IBM/carbon-designers I believe we’ll create theme tokens that applies to system-wide and go away from feature-specific color tokens in future - Is it correct?

Reply:
Ideally we'd use the tokens that we have already for colors, but for the shell work we have this set of tokens for now (can check in our squad channel to see Anna's post with this set).